modules/java/src/main/java/org/syslog_ng/LogThrDestDrvConsts.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019 Balabit
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+package org.syslog_ng;
+public abstract interface LogThrDestDrvConsts {
```

Comment:
* use [java enums](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html) and
* use [static imports](https://docs.oracle.com/javase/1.5.0/docs/guide/language/static-import.html)
(I'd call it `WorkerInsertResults`.)
You can import enum members statically, eg.: 
```java
import static org.syslog_ng.WorkerInserResultCodes.*;
```
In this way you can use these consts the same way then currently but without this weird interface-usage.

Reply:
I resolve this because in the end I could eliminate the whole class.